### ISSUE TITLE
cli "put back" support

### ISSUE THREAD

**Issue by mafintosh** [sindresorhus/osx-trash#1]
This module is awesome. Was thinking about writing something like this yesterday and then I saw you already did it which is even better. Is there anyway to "untrash" a file from the command line? Like:

```
trash --recover my-trashed-file.txt
```

**Comment by sindresorhus** [comment]
Yeah, that would be useful. Not sure if possible or how though. Unless there's a convenient API I'm missing it would basically have to go through the trash index(?) and find the file pointing back to the one you specify and then move it back.

**Comment by interglobalmedia** [comment]
Hi,

I first heard about your npm trash package but don't want to install a feature that depends on npm. Then I saw this. I am on El Capitan. Where would I install since I am on El Capitan? And what exactly is the command for moving something into the trash from Command Line? Thanks!

**Comment by russelldavis** [comment]
A note to consider about going through the trash index (.DS_Store): Finder takes a while before flushing changes to that file, so that approach won't work with recently trashed files.

The only reliable way to support a Put Back feature, short of doing everything via applescript commands to Finder (which is slow and doesn't work well with `sudo`), is to store your own metadata of the original location.

Fortunately that's pretty easy to do with `xattr`. I wrote a pair of CLI commands (`trash` and `putback`) that do it that way -- see https://github.com/russelldavis/trash101

**Comment by y-vectorfield** [comment]
I consider the following idea.
When the file is deleted using this tool, its absolute path and date of deletion are recorded.
If two or more files have the same name, it would be a good idea to restore the file with the most recent deletion date or to enumerate them and let the user choose which one to fix.

**Comment by veloxo** [comment]
"Put Back" seems to be fixed in Sequoia 15.1, afaict!

**Comment by exsesx** [comment]
What about just `mv ~/.Trash/somefile.txt .`? It could be more convenient, but still.
